ethereum/referencetests/build.gradle

```diff
@@ -32,6 +32,9 @@ dependencies {
   implementation project(':services:kvstore')
 
   implementation 'com.fasterxml.jackson.core:jackson-databind'
+  implementation 'com.google.code.findbugs:jsr305'
```

Comment:
Looks like you can get rid of JSR305, here as well as in versions.gradle.
https://app.circleci.com/pipelines/github/hyperledger/besu/5943/workflows/f4e50074-c111-46e0-ac0c-97fdfb3036bd/jobs/28696